ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -1021,6 +990,7 @@ const updateSkillDerivedForId = function (section, row_id) {
             Number(values[base]) +
             disciplineexpertise_n +
             (expertise_v === 'ST' ? -1 :
+              expertise_v === 'B' ? -1 :
               expertise_v !== '--' ? Number(expertise_v) :
```

Comment:
Generally, nested ternaries are confusing. As a follow up, I think we should split this up to increase clarity.
Perhaps a clean up follow up.

Reply:
I think of nested ternaries that are laid out like this as the equivalent of case statements, which I'm familiar with, and so don't find confusing.
What would a split up version look like?

Reply:
I'd have to look at it, but I'd likely extract named variables. Often, this tends to give the code more context and readability as well.
That being said, this is probably fine for the code base.

Reply:
Is this an improvement on the ability computation?

Reply:
I think nested ternaries will always be a problem, and are generally linted against.
https://eslint.org/docs/latest/rules/no-nested-ternary

Code formatters will auto-format things to look worse or better, so we can't rely on formatting to keep things managed.

The typical directions that I tend to go towards is a helper function with either switch case or if returns.
This also helps isolate code and improves encapsulation for testing purposes.

That all being said, this isn't code for production (i.e. not work 😸 )